interface-definitions/system-login.xml.in

```diff
@@ -36,6 +36,68 @@
                     </properties>
                     <defaultValue>!</defaultValue>
                   </leafNode>
+                  <node name="otp">
+                    <properties>
+                      <help>2FA OTP authentication parameters</help>
+                    </properties>
+                    <children>
+                      <leafNode name="key">
+                        <properties>
+                          <help>Token Key Secret key for the token algorithm (see RFC 4226)</help>
+                          <valueHelp>
+                            <format>txt</format>
+                            <description>OTP key (base32 encoded secret)</description>
+                          </valueHelp>
+                          <constraint>
+                            <regex>[A-Z2-7]{20,10000}</regex>
+                          </constraint>
+                          <constraintErrorMessage>Key must only include base32 characters and be at least 26 characters long</constraintErrorMessage>
+                        </properties>
+                      </leafNode>
+                      <leafNode name="rate_limit">
+                        <properties>
+                          <help>Number of attempts. Limit logins to N per every M seconds</help>
+                          <valueHelp>
+                            <format>u32:1-10</format>
+                            <description>Number of attempts. Limit logins to N per every M seconds</description>
+                          </valueHelp>
+                          <constraint>
+                            <validator name="numeric" argument="--range 1-10"/>
+                          </constraint>
+                          <constraintErrorMessage>Number of login attempts must me between 1 and 10</constraintErrorMessage>
+                        </properties>
+                        <defaultValue>3</defaultValue>
+                      </leafNode>
+                      <leafNode name="rate_time">
+                        <properties>
+                          <help>Time interval. Limit logins to N per every M seconds</help>
+                          <valueHelp>
+                            <format>u32:15-600</format>
+                            <description>Time interval. Limit logins to N per every M seconds</description>
+                          </valueHelp>
+                          <constraint>
+                            <validator name="numeric" argument="--range 15-600"/>
+                          </constraint>
+                          <constraintErrorMessage>Rate limit time interval must be between 15 and 600 seconds</constraintErrorMessage>
+                        </properties>
+                        <defaultValue>30</defaultValue>
+                      </leafNode>
+                      <leafNode name="window_size">
```

Comment:
We do not use underscores in CLI nodes. Please change to hyphen.

Reply:
Fixed